packages/angular/projects/oneschema/src/test.ts

```diff
@@ -8,20 +8,8 @@ import {
   platformBrowserDynamicTesting
 } from '@angular/platform-browser-dynamic/testing';
 
-declare const require: {
```

Comment:
was this meant to be changed?

Reply:
it was all done by angular automatically tho.. so i guess so.....